src/lib.rs

```diff
@@ -93,6 +93,13 @@ pub enum EmbeddingModel {
     ParaphraseMLMiniLML12V2,
     /// v1.5 release of the small Chinese model
     BGESmallZHV15,
+    /// Small model of multilingual E5 Text Embeddings
+    MultilingualE5Small,
+    /// Base model of multilingual E5 Text Embeddings
+    MultilingualE5Base,
+    // Large model is something wrong, model.onnx size is only 546kB
+    // /// Large model of multilingual E5 Text Embeddings
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_clarification):** Consider clarifying the status of the MultilingualE5Large model in the code.

It's helpful for future maintenance to include a more detailed comment on why the Large model is commented out and under what conditions it might be revisited or fixed.

```suggestion
    // The Large model of multilingual E5 Text Embeddings is currently not included due to an issue with its ONNX file size (only 546kB), 
    // indicating a potential export or conversion problem. This model variant should be revisited once the underlying issue is resolved 
    // or if a correctly sized model file becomes available.
```